src/_zkapauthorizer/replicate.py

```diff
@@ -237,24 +239,33 @@ def enable_replication(self) -> None:
         """
         self._replicating = True
 
-    def snapshot(self) -> bytes:
+    def iterdump(self) -> Iterable[str]:
```

Comment:
Proxy `iterdump` so the stand-alone `snapshot` function can work on this kind of connection.